### ISSUE TITLE
Long scan times with cut and paste 

### ISSUE THREAD

**Issue by jessestn** [ExLibrisGroup/SpineOMatic#29]
When we scan a barcode in from a barcode scanner, everything works normally and it pulls up a call number almost instantly.
If we cut and paste a barcode from a file into the scan field, the search takes much, much longer and sometimes even hangs. The search time is very variable but always longer than just scanning it in. 

The barcodes are in a plaintext file generated by our handheld barcode scanner but other manually generated files and ones that have come from a CSV file seem to do it as well. 

I don't remember ever noticing this before, but I went back to 8.1.0 and it does it.

**Comment by mgobat** [comment]
Hmmm...that's interesting.  Even more odd is if, after pasting the barcode, you move your mouse pointer off the app before hitting <Enter>, it works fast again.  @jessestn , do you see that behavior, too?

**Comment by jessestn** [comment]
Yes, I just tried it. If the mouse pointer isn't over the app when enter, it seems to go about the normal speed. 
Also, I didn't mention it before, but if you use the drop-down to "re-scan" a previously scanned barcode it also always took much longer, but moving the mouse pointer seems to fix that as well.

Weird.

**Comment by mgobat** [comment]
I've replicated the problem back to 7.0. Will look into it for the next version.
